data/profiles/taskrunner.py

```diff
@@ -1,61 +1,153 @@
-#!/usr/bin/python
-
 # Copyright 2016, EMC, Inc.
 
 """
 Download and execute python scripts from the RackHD tasks api in a loop
 """
 
-import requests
 import json
 import sys
 from time import sleep
 
+class Cisco(object):
+
+    def __init__(self):
+        pass
+
+    @staticmethod
+    def download_task_data():
+        """
+        Download a python script from the tasks API
+        """
+        _task_data = urllib2.urlopen(TASKS_URI).read()
+        poc_log("Downloading task data")
+        poc_log("_task_data: {0}".format(_task_data))
+
+        return json.loads(_task_data)
+
+    @staticmethod
+    def download_script(_downloadUrl):
+        """
+        Download a python script from downloadUrl
+        """
+        url = urllib2.urlparse.urljoin('http://<%=server%>:<%=port%>', _downloadUrl)
+        poc_log("Downloading script at {}".format(url))
+        script = urllib2.urlopen(url).read()
+        poc_log("{0}".format(script))
+        with open('script.py', 'w') as rackhd_script:
```

Comment:
@yyscamper I believe I did it this way because there wasn't a way to reference the original script name without parsing the URL value, and it's only stored temporarily anyways so I didn't think there was any harm to a generic reference.